libbpfgo.go

```diff
@@ -296,34 +296,69 @@ func errptrError(ptr unsafe.Pointer, format string, args ...interface{}) error {
 	return fmt.Errorf(format+": %v", args...)
 }
 
-func NewModuleFromFile(bpfObjFile string) (*Module, error) {
+func NewModuleFromFile(bpfObjPath string) (*Module, error) {
+	return NewModuleFromFileBtf("", bpfObjPath)
+}
+
+func NewModuleFromFileBtf(btfObjPath string, bpfObjPath string) (*Module, error) {
```

Comment:
should be `NewModuleFromFileBTF` (inconsistent with https://github.com/aquasecurity/libbpfgo/pull/53/ and with golang naming convensions)

Reply:
https://github.com/aquasecurity/libbpfgo/pull/54